integration-tests/contract-pair-as-key.spec.ts

```diff
@@ -76,7 +76,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
         done();
       })
 
-    test('originate contract and init storage with pair as key in map', 2, async (done: () => void) => {
+    test('Verify contract.originate for a contract with pair as a key in map ', 2, async (done: () => void) => {
```

Comment:
Might worth mentioning that it is the init property is used in the test instead of the storage property in the precedent test.